### ISSUE TITLE
Branch: energyBalancePhetio

### ISSUE THREAD

**Issue by jbphet** [phetsims/greenhouse-effect#85]
This is a branch where some attempts to fix a problem with state for the energy rates were made.  @zepumph and I worked on it together but couldn't quite get it working before a meeting started, so it's going here for further investigation.  The original issue is #80.

**Comment by zepumph** [comment]
Ok, there were two problems, one was @jbphet's fault (forgot to supply a phetioType), and one was my fault (using the stateSchema to applyState has a bug, see https://github.com/phetsims/tandem/issues/245#issuecomment-921065520.

@jbphet, I think you can proceed with merging this branch into master and testing.

**Comment by zepumph** [comment]
After discussion with @jbphet, the above commit wasn't enough for the full fix. Details over in #80. I merged this branch into master, but I'm a bit nervous because this isn't my sim. I'm going to keep this branch open until @jbphet get's back and can take a lay of the land. Most likely he will just delete the branch though.

**Comment by jbphet** [comment]
I reviewed the merge commit, specifically https://github.com/phetsims/greenhouse-effect/commit/bdc21f7278337a1eaf09100a2f1ec27d879803da, and it at least roughly makes sense to me.  I'm still not super clear on when to use STATE_SCHEMA and when to create specific methods like `applyState`, but I'm thinking that understanding will come as I gain more experience with this (plus I think it's still evolving a bit).  I tested several cases of the problem described in #80, and QA did too, and everything seemed to work well.  All in all, I think it's safe at this point to delete the branch.
